app/Http/Controllers/AdditionController.php

```diff
@@ -53,7 +54,12 @@ public function store(StoreAdditionRequest $request)
     {
         //
         $accounting_system_id = $this->request->session()->get('accounting_system_id');
-         for($i = 0; $i < count($request->employee); $i++)
+        // Disable create if payroll is generated for current accounting period.
+        $payroll = Payroll::where('accounting_system_id', $accounting_system_id)->get();
+        if(!$payroll->isEmpty())
+        return redirect()->back()->with('danger', 'Payroll already created! Unable to generate new addition in current accounting period.');
```

Comment:
I afraid this won't allow new additions (and others) once a single payroll is generated. This was supposed to block only when within that period. Will issue a fix.

Reply:
Fixed.

Blocked because payroll is generated for Accounting Period 1 (Jan 1-31)
![image](https://user-images.githubusercontent.com/32955000/188251284-3130eb07-5385-481c-a220-939245d36dfa.png)
Succeeding entries (Sept) will push through since a payroll hasn't generated within Accounting Period 9 (Sept 1-30)
![image](https://user-images.githubusercontent.com/32955000/188251295-c0bb6c68-2505-4e2f-ad80-9cfea138c59a.png)